### ISSUE TITLE
`append_scylla_args` in Jenkins jobs for SCT / longevity tests

### ISSUE THREAD

**Issue by kbr-scylla** [scylladb/scylla-cluster-tests#7653]
Sometimes it's useful to run SCT tests with additional parameters passed to Scylla -- for example, recently I asked @soyacz to reproduce certain failures while enabling higher-level logging for Raft by passing `--logger-log-level raft=trace:raft_topology=trace:group0_raft_sm=trace` to Scylla: https://github.com/scylladb/scylla-enterprise/issues/3820#issuecomment-2167699606, https://github.com/scylladb/scylladb/issues/18821#issuecomment-2165880053, IIUC @soyacz did it by modifying SCT on his fork and running the job with it.

Perhaps a good investment would be to extend the regular Jenkins jobs we use with something like APPEND_SCYLLA_ARGS parameter, which would then be passed to Scylla.

This would allow e.g. choosing a different set of loggers for which we enable higher logging levels for debugging some tough issues, not necessarily related to Raft. Or generally change the Scylla configuration however we want for certain runs.

**Comment by fruch** [comment]
the only problem with this one, is if the test was using this configuration, it wouldn't append to it as one might expect.
we'll need to get it figured out, but regardless of that it would be useful option for jenkins pipelines

**Comment by soyacz** [comment]
maybe we could introduce `scylla_logger_settings` param? 
This should be appendable to `append_scylla_args`

**Comment by fruch** [comment]
> maybe we could introduce `scylla_logger_settings` param? This should be appendable to `append_scylla_args`

logging is just an example, there could be more things one might want to enable from the command line, I don't think redefining all of them in SCT would be of much help.

we should make sure `append_scylla_args` is mergeable, and can be define multiple times, like we recently fixed `append_scylla_yaml` 

it's a bit more trick, since we'll need to flush out duplications (we have such code in ccm)

**Comment by soyacz** [comment]
'mergable' will work on .yaml configs level (we should do it), but I don't see how it could work with jenkins where we pass params with env variables that override config. 
`pipeline_append_scylla_args` could be used to indicate we append to ones generated by params.

**Comment by roydahan** [comment]
Now that we have the ability to change any parameter, is this still relevant?

**Comment by dimakr** [comment]
> maybe we could introduce `scylla_logger_settings` param? This should be appendable to `append_scylla_args`

Appendability of additional parameters, which this change adds, is a needed thing indeed.
But for customizing scylla loggers SCT already has handling in `sdcm/provision/scylla_yaml/scylla_yaml.py`, e.g. we ca pass custom logging params in a test config as:
```
append_scylla_yaml:
   internode_compression_enable_advanced: true
   internode_compression_zstd_max_cpu_fraction: 0.05
   logger_log_level:
     advanced_rpc_compressor: debug
```
or pass it in `extra_environment_variables` as `export SCT_APPEND_SCYLLA_ARGS='{"logger_log_level": {"advanced_rpc_compressor": "debug"}}'`

**Comment by fruch** [comment]
we now have this option available from #9554
